code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -15,6 +15,7 @@
 	brute_mod = 0.8		//About as tanky to brute as a Unathi. They'll probably snap and go feral when hurt though.
 	burn_mod =  1.15	//As vulnerable to burn as a Tajara.
 	can_fly = 1 //They have wings by default.
+	var/base_species = "Tajaran" //They have the taj sprite.
```

Comment:
... they have their own sprite. Slightly different head and shading to Tajara.

Reply:
Oh. Hmm. Will change it to "Xenochimera"

Reply:
Resolved. Made it so Xenochimera are able to choose xenochimera as their icon sprite.